src/vs/workbench/contrib/webview/browser/webviewElement.ts

```diff
@@ -370,7 +370,7 @@ export class IFrameWebview extends Disposable implements Webview {
 		const element = document.createElement('iframe');
 		element.name = this.id;
 		element.className = `webview ${options.customClasses || ''}`;
-		element.sandbox.add('allow-scripts', 'allow-same-origin', 'allow-forms', 'allow-pointer-lock', 'allow-downloads');
+		element.sandbox.add('allow-scripts', 'allow-same-origin', 'allow-forms', 'allow-pointer-lock', 'allow-downloads', 'allow-popups');
```

Comment:
seems that there are actually two `iframes`? I think this is relatively new? But the outer `iframe` has all the sandbox permissions granted always, the inner one has the conditional on the passed options.

Reply:
Yes that's correct. The outer iframe is the one we manage while the inner one has content from extensions